modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/podcast/EpisodeListAdapter.kt

```diff
@@ -54,6 +54,8 @@ class EpisodeListAdapter(
     val swipeButtonLayoutFactory: SwipeButtonLayoutFactory,
 ) : ListAdapter<BaseEpisode, RecyclerView.ViewHolder>(PLAYBACK_DIFF) {
 
+    private lateinit var podcastEpisode: PodcastEpisode
```

Comment:
⚠️ `podcastEpisode` is not being set anywhere. Accessing `podcastUuid`  from a null object will cause a crash. Try loading episodes from the filters tab for example.